### PR TITLE
Update readme with absolute URLs

### PR DESCRIPTION
**Pull request by omspacecode**
## Description

Update the all Relative URLs  to Absolute URLs. 

_Screenshot_
If relevant, add a screenshot or two of the changes you made.

### CODE REVIEW

File: packages/react/README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Builder.io React SDK
 
-NOTE: If you want to register your React Server Components as custom components in Builder, you will need to use our experimental NextJS SDK [here](../sdks/output/nextjs).
+NOTE: If you want to register your React Server Components as custom components in Builder, you will need to use our experimental NextJS SDK [here](https://github.com/BuilderIO/builder/tree/main/packages/sdks/output/nextjs).
```
Comment (samijaber) [inline packages/react/README.md:3]: I think you only need to replace the ../ with packages/

The rest is not needed

Comment (omspacecode) [reply packages/react/README.md:3]: That would result in below URL which seems incorrect
NG: https://github.com/BuilderIO/builder/blob/packages/sdks/output/nextjs
OK: https://github.com/BuilderIO/builder/tree/main/packages/sdks/output/nextjs

Comment (omspacecode) [reply packages/react/README.md:3]: I am checking if adding main/packages/ , would work for all.
OK: https://github.com/BuilderIO/builder/blob/main/packages/sdks/output/nextjs

Comment (omspacecode) [reply packages/react/README.md:3]: Adding main/packages/ would work for some. 
I see some absolute URLs used before.

Comment (samijaber) [reply packages/react/README.md]: sorry those are broken. You might need to add a missing `/` to the start. Right now the generated URL is:

https://github.com/BuilderIO/builder/blob/om-dev/packages/react/packages/sdks/output/react

Comment (samijaber) [reply packages/react/README.md]: I'm sorry for confusing you. I think the original absolute URL is the way to go ultimately. My suggestion was off! it won't work properly in npm.

Comment (omspacecode) [reply packages/react/README.md]: No worries, I was unsure myself of whether or not the links would work on both Github and npm. 
I will revert back to absolute URLs.

Comment (omspacecode) [reply packages/react/README.md:3]: Reverted back to absolute URLs.

**Review by samijaber — COMMENTED** [review]
Thanks @omspacecode :pray

**Pull request merged**